docs/exernal-services.md

```diff
@@ -0,0 +1,206 @@
+# Summary
+There are 5 types of external services supported by the Portal:
+- runtime environments
+- authoring environments
+- teacher and student reports or dashboards
+- researcher reports
+- SSO clients
+
+# Runtime Environments
+A runtime environment is a site that handles students running a resource. In some cases the authoring environment and the runtime environment will be the same. The main way the Portal works with a runtime environment is through ExternalActivities. Additionally the Portal has Tools. An ExternalActivity can be associated with one Tool. The Tool provides a way for multiple ExternalActivities to have similar behavior.
+
+Authors create these ExternalActivities (called Resources in the UI).
+
+Teachers find ExternalActivities and assign them to their class. This assignment is called an Offering in the code. In LTI terminology this is the Offering is the same as a Resource Link.
```

Comment:
I think the last sentence should say "In LTI **terminology the Offering** is the same as a Resource Link.", right?